src/Raven.Server/Documents/AI/AiAssistant/Handlers/AiAssistantHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Threading.Tasks;
+using Raven.Server.Documents.AI.AiAssistant.Handlers.Processors;
+using Raven.Server.Routing;
+using Raven.Server.Web;
+
+namespace Raven.Server.Documents.AI.AiAssistant.Handlers;
+
+public class AiAssistantHandler : ServerRequestHandler
+{
+    [RavenAction("/assistant/assist", "POST", AuthorizationStatus.ValidUser, EndpointType.Read)]
+    public async Task Assist()
+    {
+        using (var processor = new AiAssistantAssistProcessor(this))
+            await processor.ExecuteAsync();
+    }
+    
+    [RavenAction("/assistant/give-consent", "POST", AuthorizationStatus.ValidUser, EndpointType.Read)]
```

Comment:
I wonder if consent should be granted by every user or just the Operator and up.

Also this isn't endpoint type Read.